src/rgw/rgw_sal_motr.cc

```diff
@@ -457,6 +457,52 @@ int MotrUser::store_user(const DoutPrefixProvider* dpp,
 
 int MotrUser::remove_user(const DoutPrefixProvider* dpp, optional_yield y)
 {
+  // Remove user info from cache
+  // Delete access keys for user
+  // Delete user info index
+  // Delete user index
```

Comment:
It should be "Delete user from user index"

Reply:
Done